### ISSUE TITLE
How to get the same psnr in paper?

### ISSUE THREAD

**Issue by Jambalaya11** [alterzero/RBPN-PyTorch#27]
Hi, I trained the model on dataset vimeo-90k, nFrame=7, patch-size=64, future-frame=True, after about 20 epochs(four days; 8 gpus; bs=8), the loss is around 0.009.  However when I test the model, I can't get the awesome result in the paper. Vid4 walk 28.12 (paper=30.70), vimeo slow test data 31.91(paper=34.18)。  Should I wait for more epochs? I will be appreciated if you could give more training or testing detail.

**Comment by splinter21** [comment]
In main.py, default epoches is 150...

**Comment by Jambalaya11** [comment]
> In main.py, default epoches is 150...

OK.. Thank a lot

**Comment by alterzero** [comment]
I think at least you should train the network with 80 epochs.

**Comment by gittigxuy** [comment]
@Jambalaya11,@alterzero,how to train or test the model to use multi-GPU,I saw that you use gpu_lists[0] to read data,thus,it only use 1 gpu,how could I fix it to use multi-GPU?
